### ISSUE TITLE
Markdown processing of the spec converts a number into an ordered list

### ISSUE THREAD

**Issue by elyscape** [getdnsapi/getdns-website#4]
The final paragraph of [section 3.3 of the spec][section-3.3] reads as follows:

>The use of this extension can conflict with the values in the DNS context. For example, the default for an OS might be a maximum payload size of 65535, but the extension might specify
>1. In such a case, the API will honor the values stated in the extension, but will honor the values from the DNS context if values are not given in the extension.

This doesn't appear to make sense, and indeed isn't what was intended. The [processed source of that line][source] reads as follows:

```markdown
The use of this extension can conflict with the values in the DNS context. For example,
the default for an OS might be a maximum payload size of 65535, but the extension might specify
1550. In such a case, the API will honor the values stated in the extension, but will honor the
values from the DNS context if values are not given in the extension.
```

The spec is being parsed as Markdown, which means that a line beginning with a number followed by a period and a space is an entry in a numbered list. The good news is that this is a pretty easy fix: escaping the period with a backslash should prevent it from being parsed as a numbered list. Like so:

```markdown
1550\. In such
```

[section-3.3]: https://getdnsapi.net/documentation/spec/#33-setting-up-opt-resource-records
[source]: https://github.com/getdnsapi/getdns-website/blob/e4d32dd/content/documentation/spec/contents.lr#L747-L750

**Comment by wtoorop** [comment]
Good catch!  Thanks for that.

**Comment by wtoorop** [comment]
Resolved per commit f0e0a8a60cfc3b484eaa841ad07a402f18e273fd
Thanks again
